clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -45,6 +45,21 @@
 
 namespace {
 
+using namespace dynamorio::drmemtrace;
+
+#ifdef X86
+#    define REG1 DR_REG_XAX
+#    define REG2 DR_REG_XDX
+#elif defined(AARCHXX)
+#    define REG1 DR_REG_R0
+#    define REG2 DR_REG_R1
+#elif defined(RISCV64)
+#    define REG1 DR_REG_A0
+#    define REG2 DR_REG_A1
+#else
+#    error Unsupported arch
+#endif
```

Comment:
This is duplicated verbatim from raw2trace_unit_tests.cpp: could we move this to memref_gen.h and have raw2trace_unit_tests include that to share it?